test/archive.cpp

```diff
@@ -242,12 +242,12 @@ TEST(ZimArchive, validate)
 
   EXPECT_BROKEN_ZIMFILE(
     "./data/invalid.outofbounds_first_title_entry.zim",
-    "Invalid title index entry\n"
+    "Invalid title index entry.\n"
```

Comment:
This commit better be squashed into the one bringing this change.

Reply:
Done